src/app/zap-templates/zcl/data-model/chip/access-control-cluster.xml

```diff
@@ -156,16 +156,11 @@ limitations under the License.
       <access op="read" privilege="administer"/>
     </event>
 
-    <event code="0x0002" name="AccessRestrictionEntryChanged" priority="info" side="server" isFabricSensitive="true" optional="true">
-      <description>The cluster SHALL send AccessRestrictionEntryChanged events whenever its ARL attribute data is changed by the device maker.</description>
-      <access op="read" privilege="administer"/>
-    </event>
-
-    <event code="0x0003" name="FabricRestrictionReviewUpdate" priority="info" side="server" isFabricSensitive="true" optional="true">
+    <event side="server" code="0x0002" name="FabricRestrictionReviewUpdate" priority="info" isFabricSensitive="true" optional="true">
       <description>The cluster SHALL send FabricRestrictionReviewUpdate events to indicate completion of a fabric restriction review.</description>
       <field id="0" name="Token" type="int64u"/>
-      <field id="1" name="Instruction" type="long_char_string" isNullable="true" length="512"/>
-      <field id="2" name="RedirectURL" type="long_char_string" isNullable="true" length="256"/>
+      <field id="1" name="Instruction" type="long_char_string" optional="true" length="512"/>
+      <field id="2" name="ARLRequestFlowUrl" type="long_char_string" optional="true" length="256"/>
```

Comment:
This matches the spec as it stands today, but why did the spec use lower-case "rl" here?  Most URLs in the spec use "URL" with all-caps in the name...

I feel like we should align with normal spec usage and have ARLRequestFlowURL both here and in the spec.

Reply:
Created #35418